catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -71,6 +72,7 @@
 	private String callingActivity;
 	private ProgressDialog progressDialog;
 	private Intent resultIntent = new Intent();
+	private final String PACKAGE_NAME_WHATSAPP = "com.whatsapp";
```

Comment:
ok i just checked the test run, the warning comes from line 75:

```
Member name not following naming convention - Name 'PACKAGE_NAME_WHATSAPP' must match pattern '^(([a-z]{2})|([x-z][A-Z]))[a-zA-Z0-9]*$'.

Validates identifiers for non-static fields.
```

but instead of changing the name I'd rather make the variable static, since it doesn't need to be instanced ;)